src/DipStrikeTool.hpp

```diff
@@ -79,15 +85,13 @@ class DipStrikeTool : public IVistaOpenGLDraw, public cs::core::tools::MultiPoin
   VistaBufferObject      mVBO;
   VistaGLSLShader        mShader;
 
-  double    mOriginalDistance = -1.0, mSize;
+  double    mOriginalDistance = -1.0, mSize{};
   glm::vec3 mNormal = glm::vec3(0.0), mMip = glm::vec3(0.0);
-  float     mOffset, mSizeFactor = 1.5f, mOpacity = 0.5;
+  float     mOffset{}, mSizeFactor = 1.5F, mOpacity = 0.5;
 
   int mScaleConnection = -1;
 
-  static const int         RESOLUTION;
-  static const std::string SHADER_VERT;
-  static const std::string SHADER_FRAG;
+  static const int RESOLUTION;
```

Comment:
Why is RESOLUTION still a class member while the others have been moved to the source file?